src/main/java/io/jenkins/plugins/ml/IPythonBuilder.java

```diff
@@ -130,7 +142,9 @@ public String getCode() {
 
     enum FileExtension {
         ipynb,
-        json
+        json,
+        py,
+        txt
```

Comment:
@Loghijiaha are you also looking at adding support for .csv files?

Reply:
Its a good idea @shivaylamba. So far, my concern was about the code and notebook. I will take care of data set files in the next coding phase.

Reply:
Alright Loghi